unit/Makefile

```diff
@@ -16,14 +17,14 @@ LIBS += ../src/ansi-c/ansi-c$(LIBEXT) \
         ../src/cpp/cpp$(LIBEXT) \
         ../src/json/json$(LIBEXT) \
         ../src/linking/linking$(LIBEXT) \
-        ../src/util/util$(LIBEXT) \
-        ../src/big-int/big-int$(LIBEXT) \
-        ../src/goto-programs/goto-programs$(LIBEXT) \
         ../src/pointer-analysis/pointer-analysis$(LIBEXT) \
         ../src/langapi/langapi$(LIBEXT) \
         ../src/assembler/assembler$(LIBEXT) \
         ../src/analyses/analyses$(LIBEXT) \
+        ../src/goto-programs/goto-programs$(LIBEXT) \
         ../src/solvers/solvers$(LIBEXT) \
+        ../src/util/util$(LIBEXT) \
+        ../src/big-int/big-int$(LIBEXT) \
```

Comment:
Ah, that's because that's not what `LIBS` is to be used for. All the entries currently placed in `LIBS` in `unit/Makefile` ought to go in `OBJS`. The code base uses `LIBS` only for libraries outside the code base. (Yes, I do agree this isn't quite the perfect set up...)